pca9685:
  pwm:
    bits_per_period: 4095 
    frequency: 50.0
    frequency_measured: 50 # 51.6 is the former value, but we dont know why

i2c:
  pca9685:
    address: 0x40
    bus: 1
